posh-git.psm1

```diff
@@ -1,5 +1,14 @@
+param([switch]$NoVersionWarn = $false)
+
 if (Get-Module posh-git) { return }
 
+if ($PSVersionTable.PSVersion.Major -lt 3 -and !$NoVersionWarn) {
+    Write-Warning ("posh-git support for PowerShell 2.0 is deprecated; you have version $($Host.Version).`n" +

```

Comment:
Let's use `$($PSVersionTable.PSVersion)` instead of `$($Host.Version)`, since the latter will only match for the standard host (not in the NuGet console, for example).